src/index.ts

```diff
@@ -65,24 +71,33 @@ program
         "-c, --category [category]",
         "The Category name that best fits this block"
     )
-    .action(async ({ name, category }) => {
-        const nameInput = typeof name !== "function" ? name : null;
-        const categories = await getCategoryNames();
-
-        if (category) {
-            publish(nameInput, category, categories);
+    .option(
+        "-m, --major-version [majorVersion]",
+        "Publish a new major version of this block"
+    )
+    .action(async ({ name, category, majorVersion }) => {
+        if (majorVersion) {
+            newMajorVersion();
         } else {
-            inquirer
-                .prompt({
-                    choices: categories,
-                    message: "Select the Category that best fits this block:",
-                    name: "categoryFromList",
-                    type: "list",
-                })
-                .then((val: any) => {
-                    const { categoryFromList } = val;
-                    publish(nameInput, categoryFromList);
-                });
+            const nameInput = typeof name !== "function" ? name : null;
```

Comment:
can `name` be a function?

Reply:
I guess this originally wanted to make sure it wasn't lol